Updated setup instructions (#51)
carpentries-incubator/docker-introduction: setup.md

### Files to download

Download the [`docker-intro.zip`]({{ page.root }}/files/docker-intro.zip) file. _This file can alternatively be downloaded from the `files` directory in the [docker-introduction GitHub repository](https://github.com/carpentries-incubator/docker-introduction/)_.

Move the downloaded file to your Desktop and unzip it. It should unzip to a folder called `docker-intro`. 

### Software to install: Docker

> Docker can be installed on Windows 10, macOS and Linux. However, in some cases, installing Docker on your laptop may, unfortunately, not be completely straightforward if you do not have significant technical experience. We have helpers on hand that have worked their way through the install process but be prepared for some troubleshooting.

Reviewer: ```suggestion
Docker can be installed on Windows 10, macOS and Linux. However, in some cases, installing Docker on your laptop may, unfortunately, not be completely straightforward. We have helpers on hand during the workshop that have worked their way through the install process but be prepared for some troubleshooting.
```

Please try to install the appropriate software from the list below depending on the operating system that your laptop is running:

#### **Microsoft Windows**
**You must have admin rights to run docker!** Some parts of the lesson will work without running as admin but if you are unable to `Run as administrator` on your machine some elements of this workshop might not work as described.

On Windows 10, you will need to install [Docker Desktop for Windows](https://hub.docker.com/editions/community/docker-ce-desktop-windows). 

> _Previously Windows 10 Home users needed to install an alternative version of Docker - [Docker Toolbox](https://docs.docker.com/docker-for-windows/docker-toolbox/) - however, this is now deprecated. Docker Desktop for Windows can now also be installed on Windows 10 Home Edition_

Reviewer: I suggest we remove this line.  I don't think people installing for the first time will need to know this.

Reviewer: I think the section below on using toolbox as backup if you can't get docker desktop covers it well.

Author: Yes, this seems reasonable, I've removed the sentence starting "Previously Windows 10 Home users..."
